apps/examples/fs_helper/Kconfig

```diff
@@ -24,6 +24,15 @@ config ROMFS_TEST
 	---help---
 		Enable "romfs_test" application
 
+config FS_RW_TEST
+	bool "fs rw test"
+	depends on TASH
+	depends on FS_SMARTFS
+	default n
+	---help---
+		By default, if CONFIG_TASH are defined
+		Enable "fs_rw_test" application
```

Comment:
I don't understand what you want to say. `By default, if CONFIG_TASH are defined Enable "fs_rw_test" application`